modules/update-service-overview.adoc

```diff
@@ -19,7 +19,7 @@ To allow the OpenShift Update Service to provide only compatible updates, a rele
 
 [IMPORTANT]
 ====
-The OpenShift Update Service displays all recommended updates for your current cluster.  If an update path is not recommended by the OpenShift Update Service, it might be because of a known issue with the update or the target release.
+The OpenShift Update Service displays all recommended updates for your current cluster. If an update path is reported to have known issues by the OpenShift Update Service, the issues might be related to the update or the target release.
```

Comment:
I think the outgoing sentence here was more about "what if I don't see 4.y.z as recommended?".  There are more cases there:

* It's supported, but the cluster-version operator assesses at least one known issues as matching the cluster.
* It's not supported ever.  E.g. 4.10.0 straight to 4.15.0 in one hop.  Or a fresh 4.15.5 install wanting to go to 4.15.0.
* It's not in your channel yet.  E.g. 4.14 to 4.15 before openshift/cincinnati-graph-data#5012 added those to `stable-4.15`.

I think we want to keep something in here that explains that the Update Service results will exhaustively cover all the recommendations that currently match your chosen channel's semantics.  And I'm agnostic about whether we try to break down the various subsets of not-in-`availableUpdates`.